src/components/Dashboard/Dashboard.js

```diff
@@ -57,10 +56,7 @@ function Dashboard() {
         <Welcome />
       )}
 
-      <Split
-        primary={<TaskTable tasks={tasks} />}
-        secondary={<DashboardStats />}
-      />
+      <Tasks fromDashboard />
```

Comment:
What do you think of naming this `noHeader` or something like that so it's more general?

Reply:
yeah i changed that in a PR against develop in the task-list branch, the name that i used is `onlyTable`